test_suite/generic/test_generic.py

```diff
@@ -585,11 +585,12 @@ def test_cloud_cfg(self, host):
         """
         Verify file /etc/cloud/cloud.cfg is not changed
         """
-        package_to_check = 'cloud-init'
+        cloud_cfg = '/etc/cloud/cloud.cfg'
+        verify_cmd = f'rpm -Vf {cloud_cfg} | grep -e "^S.5.*{cloud_cfg}$"'
 
         with host.sudo():
-            assert host.run_test(f'rpm -V {package_to_check}'), \
-                f'There should not be changes in {package_to_check} package'
+            assert not host.run(verify_cmd).stdout, \
+                f'There should not be changes in {cloud_cfg} package'
```

Comment:
The comment does not make sense any more. `cloud_cfg` is not a package AFAICT.

Reply:
Fixed in 16aedb1cc4427de8879b287ed0db459c9af76639